content/de/cloud_native_security.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cloud Native Security
+status: Feedback appreciated
```

Comment:
The EN term "Cloud Native Security" is GA, so let's mark this one "Completed". 😊

```suggestion
status: Completed
```